docs/levels.md

```
@@ -214,14 +214,81 @@ r = composite_rating_level([(63.2, 12, 0.0),    # day
+
+```python
+import matplotlib.pyplot as plt
+from phonometry import assess_tonal_audibility
+
+# ISO 1996-2:2007 Annex C.5, Example 2 (two tones near 400 Hz):
+res = assess_tonal_audibility(tone_level=54.1, masking_noise_level=45.2,
+                              centre_frequency=430.0)
+print(res.audibility, res.adjustment)   # ΔLta ≈ 11.1 dB -> Kt = 6 dB
+res.plot()
+plt.show()
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)